src/components/main/MainContent.jsx

```diff
@@ -0,0 +1,51 @@
+function MainContent() {
+  return (
+    <div>
+      <div className="text__lg text__center">
+        <div className="new__note">
+          <button className="btn btn__primary">Create a New Note</button>
+        </div>
+        <div className="note__input">
+          <input type="text" placeholder="Title" className="input__title" />
+          <textarea
+            name=""
+            id=""
+            cols="30"
+            rows="10"
+            placeholder="Body of the Note"
+            className="input__body"></textarea>
+          <div className="text__lg input__bottom">
+            <i className="fa-solid fa-palette input__icons"></i>
+            <i className="fa-solid fa-tag input__icons"></i>
+            <i className="fa-solid fa-box-archive input__icons"></i>
+            <i className="fa-solid fa-trash input__icons"></i>
```

Comment:
this is also used below you keep it in separate file and export it instead avoid repetition

Reply:
sure